shared/main.desktop.js

```diff
@@ -53,6 +55,9 @@ class Main extends Component<void, Props, void> {
   componentDidMount () {
     if (flags.admin) window.addEventListener('keydown', this._handleKeyDown)
     ipcRenderer.send('showTray', this.props.menuBadge)
+
+    window.addEventListener('online', this.props.checkReachability)
```

Comment:
We used to do a bootstrap() on the HTML online event, so this keeps that behavior around.(This checks reachability since online only means that there is network connectivity, not that you can reach keybase.)

I think this is because on certain UI, with some network errors, there is no retry?
We could remove this and see what problems we have and fix them, or leave this in and worry about it down the road?